### PR TITLE
Issue #7 run tests

### PR DESCRIPTION
**Pull request by RomanMin**
run tests
done

### CODE REVIEW

**Review by RomanMin — COMMENTED** [review]
ok

**Review by RomanMin — COMMENTED** [review]
ok

**Review by RomanMinaevTestUser — APPROVED** [review]
ОК

**Comment by RomanMinaevTestUser** [comment]
god

**Review by RomanMinaevTestUser — COMMENTED** [review]
wait

**Review by RomanMinaevTestUser — COMMENTED** [review]
run

**Review by RomanMinaevTestUser — COMMENTED** [review]
run2

File: src/test/java/HomeWork03/SpoonaccularApiTest.java [hunk 1]
```diff
@@ -183,7 +183,7 @@ void testPostWeekPlan() {
                 .uri()
                 .expect()
                 .statusCode(200)
-                .time(lessThan(1500L))
+                .time(lessThan(2000L))
```
Comment (RomanMinaevTestUser) [inline src/test/java/HomeWork03/SpoonaccularApiTest.java:186]: run

Comment (RomanMinaevTestUser) [reply src/test/java/HomeWork03/SpoonaccularApiTest.java:186]: qw

**Review by RomanMinaevTestUser — COMMENTED** [review]
123

**Review by RomanMinaevTestUser — APPROVED** [review]

**Pull request merged**